mdk/src/main/java/com/example/examplemod/ExampleMod.java

```diff
@@ -45,8 +51,8 @@ private void setup(final FMLCommonSetupEvent event)
         LOGGER.info("HELLO FROM PREINIT");
         LOGGER.info("DIRT BLOCK >> {}", Blocks.DIRT.getRegistryName());
     }
-
-    private void doClientStuff(final FMLClientSetupEvent event) {
+    
+    public void doClientStuff(final FMLClientSetupEvent event) {
```

Comment:
You left in random white space and I do not believe there is any reason this change from private to public is needed.

Reply:
Yeah, I use class of TileEntityRenderer for testing

Reply:
You still didn't get rid of the random white space change...

Reply:
Do you mean this?
![image](https://user-images.githubusercontent.com/35610168/68790173-d98c6680-0657-11ea-8c2a-de6749622f9b.png)
It also exist in original code
![image](https://user-images.githubusercontent.com/35610168/68790212-ea3cdc80-0657-11ea-94aa-a4bfdff890bf.png)

Reply:
No, look at the dif I highlighted in this comment, I selected two lines on purpose. The line above `doClientStuff` has a random `    ` added to it.